### PR TITLE
add redux tutorial link 

### LINKED ISSUE #1944: [Add] <Want to add Redux Toolkit playlist in English >
### Site URL

https://www.youtube.com/playlist?list=PLC3y8-rFHvwiaOAuTtVXittwybYIorRB3

### Description of the above provided site

In this tutorial series on Redux Toolkit for beginners, they dive into the what, why and the how.

### Category

Frontend

### Sub Category

"react"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Tusharrgrg**
## Fixes Issue

Closes #1944

## Changes proposed

Added a Redux Toolkit in english tutorial link.

## Screenshots
I haven't set it up locally; I just added the link.

## Note to reviewers

Please consider merging this pull request.

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -114,5 +114,12 @@
     "url": "https://www.theodinproject.com/",
     "category": "frontend",
     "subcategory": "react"
+  },
+  {
+    "name": "Redux Toolkit Tutorial",
+    "description": "In this tutorial series on Redux Toolkit for beginners, we dive into the what, why and the how.",
```
Comment (CBID2) [inline database/frontend/react.json:120]: [proofreading] just a typo
```suggestion
    "description": "In this beginner-level tutorial series on Redux Toolkit, we dive into the what, why, and the how.",

```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Tusharrgrg! :) I pointed out a couple of proofreading errors in my inline comment. In the future, consider running your descriptions through Grammarly or spell check before making a pull request. Other than that, you're off to a great start! :)

**Comment by Tusharrgrg** [comment]
Hey @CBID2, thank you for pointing that out. I have now updated the description, so please take a look at it and merge it.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

File: database/frontend/react.json (hunk 1 above)
Comment (Anmol-Baranwal) [reply database/frontend/react.json]: Done!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**